runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java

```diff
@@ -167,6 +169,8 @@ public void onConfigChanged(ConfigChanged configChanged)
 			{
 				removeClanCounter();
 			}
+
+			colorIgnoredPlayers();
```

Comment:
This needs to run on the client thread. Also you could check the config key being changed.